arch/lib/Makefile

```diff
@@ -96,10 +96,11 @@ comma= ,
 kernel/_to_keep=notifier.o params.o sysctl.o \
 rwsem.o semaphore.o kfifo.o cred.o user.o groups.o ksysfs.o
 kernel/time/_to_keep=time.o
-kernel/rcu_to_keep=rcu/srcu.o rcu/pdate.o rcu/tiny.o
+kernel/rcu_to_keep=rcu/srcu.o rcu/update.o rcu/tiny.o
```

Comment:
this is probably my typo: but it means we haven't use this file for almost a couple of years, we can delete rcu/update.o from this list.